cmd/openshift-install/agent/waitfor.go

```diff
@@ -127,6 +132,13 @@ func newWaitForInstallCompleteCmd() *cobra.Command {
 				https://docs.openshift.com/container-platform/latest/support/troubleshooting/troubleshooting-installations.html`)
 				logrus.Exit(exitCodeInstallFailed)
 			}
+			if err := agentpkg.AddRouterCAToClusterCA(cluster.Ctx, config, assetDir); err != nil {
```

Comment:
@andfasano The e2e tests are failing because of a missing certificate and I could see that the `wait for` command has the step to add router CA which is missing in `agent wait for`.
I have verified that these tests are working fine with this fix.

Reply:
As mentioned before, I still found a little bit strange to address this problem just via the (read-only) `wait for`, given that in ABI is not a required command to deploy a cluster (rather than an optional one) .
In the regular non-ABI flow the `addRouterCAToClusterCA()`  func is anyhow always triggered in the `create` command (but not as an asset), thus at the end of the installation the generated kubeconfig is guaranteed to contain also the additional RouterCA.

I think that a similar behavior must be implemented for ABI as well, but rather in the `create` command, to ensure that from the very beginning the generated kubeconfig will contain the required certs (if possible). To track that, I've created the following bug https://issues.redhat.com/browse/OCPBUGS-45256